evennum.js

```diff
@@ -0,0 +1,7 @@
+function findEven(){
+    var number=window.prompt("enter number");
+    var n;
+    if(n%2==0)
+    return("true");
+    else("false");
+}
```

Comment:
1.  Change name to checkIfNumberIsEven()
2. Why var number=window.prompt("enter number");?
3. Did you test this before opening a PR?